### PR TITLE
Hide average in rubrics results #5036

### LINKED ISSUE #5036: Hide average in rubrics results
Continuation of #5015, which missed this out.

### PR DESCRIPTION
**Pull request by kanghj**
Fixes #5036 

For your review, @junhaoyap. Thanks!

### CODE REVIEW

**Comment by damithc** [comment]
Need to release ASAP.

**Comment by kanghj** [comment]
@bamboo3250, @wkurniawan07, @JosephineKwa, @LowWeiLin, @taniach, @junhaoyap, @yj-soh, any of you free to review right now?

File: src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java [hunk 1]
```diff
@@ -479,17 +479,22 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
         for (int j = 0; j < numOfRubricSubQuestions; j++) {
             StringBuilder tableBodyFragmentHtml = new StringBuilder();
             for (int i = 0; i < numOfRubricChoices; i++) {
+                String tableBodyCellValue = view.equals("student")
```
Comment (LowWeiLin) [inline src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java]: Add comments to explain the difference for student and non-student views

**Comment by LowWeiLin** [comment]
Request to add a comment. looks fine otherwise.

**Comment by kanghj** [comment]
Changes made @LowWeiLin

File: src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java [hunk 2]
```diff
@@ -479,17 +479,25 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
         for (int j = 0; j < numOfRubricSubQuestions; j++) {
             StringBuilder tableBodyFragmentHtml = new StringBuilder();
             for (int i = 0; i < numOfRubricChoices; i++) {
+                // hide numerical value of option for student's view
+                String tableBodyCellValue = view.equals("student")
```
Comment (LowWeiLin) [inline src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java]: Actually, the df.format(rubricStats[j][i] * 100) + "%" part is common. Can move it out of the condition?

**Comment by kanghj** [comment]
Changes made @LowWeiLin

**Comment by kanghj** [comment]
Anyway, even for the instructors' results page, the average computed appears to be wrong.

**Comment by damithc** [comment]
Do we need to pull the feature entirely?

**Comment by kanghj** [comment]
Yes, I think that will be the best option temporarily. @LowWeiLin, thoughts on this?

Maybe we can try to complete #4746, and fix the computation of the average that was missed out in #4915, by the next release

**Comment by damithc** [comment]
I don't mind pulling it out. It was meant to be a half-way measure anyway. We can wait till we do a proper 'assign numerical values' feature.

**Comment by kanghj** [comment]
Ok, I'll remove the averages in this pr too then.

**Comment by kanghj** [comment]
@LowWeiLin, changes made

**Comment by kanghj** [comment]
@LowWeiLin, remember to review this

**Comment by kanghj** [comment]
@damithc, merged to release as V5.72.02

**Comment by junhaoyap** [comment]
Wasn't able to get to it at such short notice - had a version to release at work at the same time!

**Comment by damithc** [comment]
@kanghj deployed hot fix V5.72.02

**Pull request merged**